### PR TITLE
E2E tests for patient document and fixing

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR add  : 
- A E2E tests for patient documents 
- Fix patient avatar style 
- Move document style to the css folder

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles could you review this PR

File: client/src/partials/patients/documents/modals/documents.modal.html [hunk 1]
```diff
@@ -30,28 +30,34 @@
...
-      <button
-        class="btn btn-danger btn-sm"
-        ng-click="$ctrl.dismiss($ctrl.file)"
-        >
-        <span class="glyphicon glyphicon-trash"></span> {{ 'FORM.BUTTONS.DISMISS' | translate }}
-      </button>
-      {{ $ctrl.file.name }}
-    </div>
+    <div style="margin-top: 10px;" class="row">
+      <div class="form-group col-xs-10"
```
Comment (jniles) [inline client/src/partials/patients/documents/modals/documents.modal.html]: This `col-xs-10` breaks on really small devices (like the Nexus 5).  You can try it in your dev tools.

I propose you set two col widths: one for large devices and one for small.  Like this: `col-md-10 col-xs-7`.  That way, the buttons do not overflow on a small device.

**Comment by jniles** [comment]
@mbayopanda, this looks pretty good.

I've made one comment, and await those changes.  After, this branch should be squashed before merging into master.

**Comment by jniles** [comment]
@mbayopanda, LGTM.  Once tests pass, I'll merge this in!

**Pull request merged**